cmd/strelaypoolsrv/gui/index.html

```diff
@@ -38,11 +44,11 @@
     <div class="container">
       <h1>Relay Pool Data</h1>
       <div ng-if="relays === undefined" class="text-center">
-        <img src="//cdnjs.cloudflare.com/ajax/libs/galleriffic/2.0.1/css/loader.gif" alt=""/>
+        <img src="https://cdnjs.cloudflare.com/ajax/libs/galleriffic/2.0.1/css/loader.gif" alt=""/>
         <p>Please wait while we gather data</p>
       </div>
       <div>
-        <div ng-show="relays !== undefined" class="ng-hide">
+        <div ng-show="relays === undefined" class="ng-hide">
```

Comment:
pretty sure this change is incorrect

Reply:
Yes it me I change that when I load the map